addons/project/static/src/js/project_form.js

```diff
@@ -172,3 +172,6 @@ export const ProjectFormView = FormDescriptionExpanderView.extend({
 viewRegistry.add('project_task_form', ProjectFormView);
 
 viewRegistry.add('form_description_expander', FormDescriptionExpanderView)
+
+// TODO: remove me in a master and create a new project view without js_class
+viewRegistry.add('project_form', FormDescriptionExpanderView);
```

Comment:
Is it not fixed with this PR: https://github.com/odoo/enterprise/pull/32188 ? 🤔

Reply:
Yes, It is fixed by that PR :star_struck: and Done the other changes. Thanks :slightly_smiling_face: